app/controllers/api/restful_controller.rb

```diff
@@ -18,7 +20,8 @@ def destroy_action
   private
 
   def current_user
-    super || LoggedOutUser.new
+    token_user = User.find(doorkeeper_token.resource_owner_id) if doorkeeper_token
```

Comment:
@gdpelican I'm still not quite clear on whether this PR should include more assurance that other API routes will work with this, or whether we should just go ahead with this, and then follow on with deeper implementations into the rest of the API routes. Do you have a sense?